Sources/Core/HTTP/HTTP.swift

```diff
@@ -224,6 +224,19 @@ public struct HTTP {
         public var bodyString: String? {
             return body.flatMap { String(data: $0, encoding: .utf8) }
         }
+
+        /// Returns a string of formatted JSON.
+        ///
+        /// - Parameters:
+        ///     - options: Options for creating the JSON data. Default is `prettyPrinted`.
+        func formattedJSON(with options: JSONSerialization.WritingOptions = [.prettyPrinted]) -> String? {
+            guard let data = body,
+                  let object = try? JSONSerialization.jsonObject(with: data, options: []),
```

Comment:
Can we do something on errors? Right here we're just swallowing problems with serialization, rather than passing the error up the chain.

We could even just make the function `throw` the errors upwards.

Reply:
Yea, I can definitely make this a `throw`ing function instead.

Reply:
Awesome!